atlp-rwanda/atlp-pulse-bn
# #54400 Refactoring user model

### PR Description
This PR refactors the user model and updates some files

### Description of tasks that were expected to be completed
- Refactor user model
- Update files import (File that may affected by user model changes)

### How has this been tested?
- Clone repo
- Checkout branch `refactor/user-model`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully

With deployed Link: <add preview and credential if they are needed>

### Track PR (issue number & link)
N/A

### Screenshots (If appropriate especially on the frontend)
N/A
